rig/place_and_route/utils.py

```diff
@@ -257,7 +254,13 @@ def build_application_map(vertices_applications, placements, allocations,
 
 
 def build_routing_tables(routes, net_keys, omit_default_routes=True):
-    """Convert a set of RoutingTrees into a per-chip set of routing tables.
+    """**DEPRECATED** Convert a set of RoutingTrees into a per-chip set of
+    routing tables.
+
+    .. warning::
+        This method has been deprecated in favour of
+        :py:meth:`rig.routing_table.routing_tree_to_tables` and
+        :py:meth:`rig.routing_table.minimise`.
```

Comment:
"E.g. most applications should use something like:

```py
from rig.routing_table import routing_tree_to_tables, minimise
tables = minimise(routing_tree_to_tables(routes, net_keys), target_lengths)
```

Where target_length gives the number of available routing entries on the chips in your SpiNNaker system (see :py:func:`~rig.routing_table.utils.build_routing_table_target_lengths`)"